src/Components/IconEditorTab/IconEditorTab.scss

```diff
@@ -136,27 +137,72 @@
   .neoletter-form-icon-editor-tab .icon-editor-preview .bs-icon {
     color: #eee;
   }
+
+  .neoletter-form-icon-editor-tab .icon-editor-preview .button-del {
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.8);
+  }
+
   #neoletter-icon-search .bs-icon {
     color: #aaa;
   }
   #neoletter-icon-search #search-input {
-    background: #333;
+    background: rgba(222, 222, 222, 0.2);
     border: 1px solid #333;
-    color: #777;
+    color: #ddd;
   }
+
+  input::placeholder {
+    color: #999 !important;
+  }
+
   .neoletter-form-icon-editor-tab .icon-container a {
     color: #eee;
     &:hover,
     &:active {
-      background: #eee;
-      color: black;
+      background: rgba(222, 222, 222, 0.2);
+      color: #fff;
       text-decoration: none;
       box-shadow:
-        0 0 3px rgba(0, 0, 0, 0.2),
-        0 0 0 2px #426698 inset;
+        0 0 4px 1px rgba(0, 0, 0, 0.1),
+        0 0 0 2px #507ab5 inset;
+      .bs-icon {
+        font-size: 14px;
+      }
     }
     &.active {
-      @include activeIcon();
+      background: rgba(222, 222, 222, 0.1);
+      color: #eee;
+      text-decoration: none;
+      box-shadow:
+        0 0 4px 1px rgba(0, 0, 0, 0.1),
+        0 0 0 2px #507ab5 inset;
+      .bs-icon {
+        font-size: 14px;
+      }
     }
   }
 }
+
+// -- Layout -- //
+
+.icon-grid {
```

Comment:
I don't know, where this is coming from, but the icon editor looks cut of:

![Screenshot 2024-09-12 at 14 34 24](https://github.com/user-attachments/assets/9c15ba76-6310-47bb-8023-5d22746d552f)

I tried it with Brave on [the preview deployment](https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://agessler-poc-icon-editor.scrivito-portal-app.pages.dev/de).

Reply:
FIXED in #[3e3b10a]

Reply:
It still looks broken

In Brave: 
<img width="1624" alt="Screenshot 2024-09-13 at 11 42 50" src="https://github.com/user-attachments/assets/01681393-ec87-4016-beb1-60b2783dc707">

In Firefox:
<img width="1624" alt="Screenshot 2024-09-13 at 11 42 40" src="https://github.com/user-attachments/assets/80f71e07-9c82-4d91-914c-98816aeb6c0e">

Reply:
Strange: on localhost I can't reproduce the issue.

Reply:
So on http://localhost:8080 it is working, but not on https://agessler-poc-icon-editor.scrivito-portal-app.pages.dev/de . Any idea why?

Reply:
Looks like some assets are not loaded or old cache files.